src/tests/ftest/soak/soak_harassers.yaml

```diff
@@ -166,6 +161,36 @@ fio_harasser:
         - 'randrw'
     oclass:
         - 'RP_2G1'
+mdtest_harasser:
+    # maximum timeout for a single job in test in minutes
+    job_timeout: 20
+    nodesperjob:
+        - 1
+        - 4
+        - 8
+    taskspernode:
+        - 4
+        - 16
+    test_dir: "/"
+    api:
+        - 'DFS'
+        - 'POSIX'
+    iteration: 1
+    num_of_files_dirs: 10000
+    pre_iter: 10
+    flags: '-L -F -N 1 -P -v'
```

Comment:
Options '-L' and '-F' will ensure the operations only happen on files which might be good for a performance test but I believe for a soak test we should include directory operations as well.
I am not quite sure how '-N 1' impacts results, but if it is something to get optimized numbers, may be we can remove that too.